### PR TITLE
Added TF test install

### PR DESCRIPTION
**Pull request by l-bat**
test_install
* Torch
    * cpu job: 38
    * gpu job: 40

### CODE REVIEW

File: tests/tensorflow/conftest.py [hunk 1]
```diff
@@ -94,3 +97,45 @@ def weekly_tests(request):
 @pytest.fixture(scope="module")
 def models_dir(request):
     return request.config.getoption("--models-dir")
+
+
+@pytest.fixture(scope="function")
+def tmp_venv_with_nncf(tmp_path, package_type, venv_type):  # pylint:disable=redefined-outer-name
```
Comment (vshampor) [inline tests/tensorflow/conftest.py:103]: This looks very much like its counterpart in torch tests. Can this be refactored to increase code reuse between framework-specific tests?

**Review by vshampor — CHANGES_REQUESTED** [review]

**Comment by l-bat** [comment]
Jenkins please retry a build

**Comment by l-bat** [comment]
Jenkins please retry a build

File: tests/tensorflow/conftest.py [hunk 2]
```diff
@@ -54,6 +55,9 @@ def pytest_addoption(parser):
     parser.addoption(
         "--models-dir", type=str, default=None, help="Path to checkpoints directory for weekly tests"
     )
+    parser.addoption(
+        "--run-install-tests", type=str, help="To run installation tests"
```
Comment (vshampor) [inline tests/tensorflow/conftest.py:59]: Make sure that CI knows about this, later on.

**Review by vshampor — APPROVED** [review]

**Pull request merged**